### ISSUE TITLE
Null Object Reference when selecting "My Mods" if no mods exist

### ISSUE THREAD

**Issue by ManDude** [larsiusprime/tdrpg-bugs#884]
![dq00](https://cloud.githubusercontent.com/assets/7569514/15803204/047595c8-2aca-11e6-9c16-5c029264235f.png)
1. Click Mods button.
2. Select My Mods.
3. Click Okay.

**Comment by ManDude** [comment]
Does NOT happen if there are mods in the Mods folder.

**Comment by larsiusprime** [comment]
Fixed in 0.9.2

**Comment by codingthat** [comment]
This isn't fixed on native Linux in DQ2:

```
Called from hxcpp::__hxcpp_main
Called from ApplicationMain::main ApplicationMain.hx line 159
Called from ApplicationMain::create ApplicationMain.hx line 71
Called from lime.app.Application::exec lime/app/Application.hx line 212
Called from lime._backend.native.NativeApplication::exec lime/_backend/native/NativeApplication.hx line 136
Called from lime._backend.native.NativeApplication::handleRenderEvent lime/_backend/native/NativeApplication.hx line 370
Called from lime.app.Event_Void_Void::dispatch lime/app/Event.hx line 179
Called from lime.app.Application::__catchError lime/app/Application.hx line 340
Called from openfl.display.Application::__dispatchErrorEvent openfl/display/Application.hx line 88
Called from openfl.events.EventDispatcher::dispatchEvent openfl/events/EventDispatcher.hx line 95
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDispatcher.hx line 237
Called from crashdumper.CrashDumper::onErrorEvent crashdumper/CrashDumper.hx line 196
Called from com.leveluplabs.tdrpg.CrashDumperTDRPG::doErrorStuff com/leveluplabs/tdrpg/CrashDumperTDRPG.hx line 124
```
